app/presenters/katello/sync_status_presenter.rb

```diff
@@ -0,0 +1,92 @@
+module Katello
+  class SyncStatusPresenter
+    include ::ActionView::Helpers::DateHelper
+    include ::Katello::TranslationHelper
+    STATUS_VALUES = { 
+                       :stopped      => _("Syncing Complete."),
+                       :error        => _("Sync Incomplete"),
+                       :never_synced => _("Never Synced"),
+                       :running      => _("Running"),
+                       :canceled     => _("Canceled"),
+                       :paused       => _("Paused")
+                    }.with_indifferent_access
+
+    class Status
```

Comment:
Can't we just update that method to access the data the right way with this change instead of introducing this 2-way access for a single method call?